packages/salesforce/data_stream/apex/manifest.yml

```
@@ -28,6 +28,14 @@ streams:
         type: bool
         multi: false
         default: false
+      - name: request_timeout
+        type: text
+        title: Request Timeout
```

```suggestion
        title: Request timeout
```

As other fields are in "sentence case" I'd suggest this ^^ and "proper case" is being currently used here. Can you make similar changes in README as well as other *.yml